### ISSUE TITLE
bundling failed after upgrade from 0.55 to 0.56

### ISSUE THREAD

**Issue by dev4xy** [facebook/react-native#20279]
I get this error after run `react-native run-android` 

> error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` from `E:\xampp\htdocs\my-app\MyApp\node_modules\react-native\Libraries\react-native\react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map

```
  "dependencies": {
    "prop-types": "^15.6.1",
    "react": "16.4.1",
    "react-native": "0.56.0",
    "react-native-elements": "^1.0.0-beta5",
    "react-native-vector-icons": "^4.6.0",
    "react-navigation": "^2.0.1"
  },
  "devDependencies": {
    "babel-jest": "23.4.0",
    "babel-preset-react-native": "5",
    "jest": "23.4.1",
    "react-test-renderer": "16.4.1"
  },
```
